helm/ph-ee-engine/templates/post-installation-job/job.yaml

```diff
@@ -55,19 +55,36 @@ spec:
           apt-get install -y kubelet kubeadm kubectl
           apt-mark hold kubelet kubeadm kubectl
 
-          export ENV_NAMESPACE={{ .Release.Namespace }}
-          echo "check-namespace"
-          echo $ENV_NAMESPACE
-          git clone --depth 1 --branch v1.4.0 https://github.com/openMF/ph-ee-env-labs.git 
-          cd ph-ee-env-labs/helm/es-secret/
-          make secrets || echo "elastic-certificates" already exists
+          # secret creation
+          git clone -b 7.17 https://github.com/elastic/helm-charts.git elastic/helm-charts
+          # sleep 300
+          cd elastic/helm-charts/elasticsearch/examples/security/
+          make secrets || echo "elastic-secrets" already exists
+          git clone -b 7.17 https://github.com/elastic/helm-charts.git elastic/helm-charts
+          cd elastic/helm-charts/kibana/examples/security/
+          make secrets || echo "kibana-secrets" already exists
+          kubectl get secret elastic-certificate-crt -n default -o yaml | sed 's/namespace: default/namespace: paymenthub/' | kubectl create -f -
+          kubectl get secret elastic-certificate-pem -n default -o yaml | sed 's/namespace: default/namespace: paymenthub/' | kubectl create -f -
+          kubectl get secret elastic-certificates -n default -o yaml | sed 's/namespace: default/namespace: paymenthub/' | kubectl create -f -
+          kubectl get secret elastic-credentials -n default -o yaml | sed 's/namespace: default/namespace: paymenthub/' | kubectl create -f -
+          kubectl get secret kibana -n default -o yaml | sed 's/namespace: default/namespace: paymenthub/' | kubectl create -f -
+          kubectl get secrets -n paymenthub
+          sleep 60
 
-          cd ../../helm/kibana-secret/
-          make secrets || echo "kibana" already exists
+          # export ENV_NAMESPACE={{ .Release.Namespace }}
+          # echo "check-namespace"
+          # echo $ENV_NAMESPACE
+          # git clone --depth 1 --branch v1.4.0 https://github.com/openMF/ph-ee-env-labs.git 
+          # cd ph-ee-env-labs/helm/es-secret/
+          # make secrets || echo "elastic-certificates" already exists
 
+          # cd ../../helm/kibana-secret/
+          # make secrets || echo "kibana" already exists
+          # kubectl get secrets -n paymenthub
+          
           #insatll netcat
           apt install -y netcat
-          until nc -vz ph-ee-zeebe-ops 80; do echo "Waiting for zeebe-ops service"; sleep 2; done;
+          # until nc -vz ph-ee-zeebe-ops 80; do echo "Waiting for zeebe-ops service"; sleep 2; done;
```

Comment:
Donot comment this line, instead use a time out based loop.
Ref: https://stackoverflow.com/questions/12321469/how-to-run-a-bash-command-until-it-is-successful-or-a-timeout-is-reached